@xen-orchestra/web-core/lib/components/ui/card-item/UiCardItem.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="ui-card-item typo p3-regular">
+    <div class="title">
+      <slot />
+    </div>
+    <div class="content text-ellipsis">
+      <slot name="content" />
+    </div>
```

Comment:
Ellipse text should have a tooltip of its contents.
I'm not very familiar with `vTooltip`, there may be a better way to do this, but the above code should work.

```suggestion
    <div v-tooltip={selector: ".content"} class="content-wrapper text-ellipsis">
      <div class="content">
         <slot name="content" />
      </div
    </div>
```

Reply:
I'm not found of adding a `<div>` just to enable the tooltip. However, I didn't manage to get the tooltip working using other methods while keeping the current layout.
After digging a bit, it is the `display: flex` on the `.content` that causes the issue.
I think `flex` is not useful here, and we can make the tooltip work without adding any extra `<div>`.
So, here is my proposed solution:
```html
<template>
  <div class="vts-card-row-key-value typo p3-regular">
    <div class="key">
      <slot name="key" />
    </div>
    <div v-tooltip class="content text-ellipsis">
      <slot name="value" />
    </div>
    <div v-if="slots.addons" class="addons">
      <slot name="addons" />
    </div>
  </div>
</template>

<script lang="ts" setup>
import { vTooltip } from '@core/directives/tooltip.directive'

const slots = defineSlots<{
  key(): any
  value(): any
  addons?(): any
}>()
</script>

<style lang="postcss" scoped>
.vts-card-row-key-value {
  display: flex;
  align-items: center;
  gap: 0.8rem;

  .key {
    min-width: 12rem;
    overflow-wrap: break-word;
    color: var(--color-neutral-txt-secondary);
  }

  .content {
    color: var(--color-neutral-txt-primary);
  }

  .addons {
    display: flex;
    gap: 0.8rem;
    margin-left: auto;
    font-size: 1.6rem;
  }
}
</style>

```
I've included the changes requested by @MathieuRA, and I also updated the names of the slots to add better readability.

It is just a proposition, feel free to try if this works well for the different use cases we can have.
As for the example in the story with the `UiTags`, since `flex` is removed, we should handle the case directly when adding elements to the slot to make it work properly:
```html
<template #value>
  <div style="display: flex; gap: 0.4rem; flex-wrap: wrap">
    <UiTag v-for="tag in tags" :key="tag" accent="info" variant="secondary" size="small">{{ tag }}</UiTag>
  </div>
</template>
```
It's just for the demo, the real implementation should use a class or a wrapper component instead of applying CSS directly to the style attribute.
As a general rule, I think the slot wrapper in a component shouldn’t add styles like `flex` and handle any "list" display. Instead, if we have to display a list of items, we could create a component for that (like a `VtsTagList` for example) that will apply the correct styles.
What do you think?